tests/test_basic.py

```diff
@@ -28,71 +29,70 @@ def test_builtin_modules(self):
         self.assertFalse(sorted(unknown_builtins))
 
 
-class TestSysModules(CurrentVersionBase):
+# class TestSysModules(CurrentVersionBase):
+#     # This relies on invocation in a clean python environment using unittest
+#     # not using pytest.
```

Comment:
I think I'd prefer to just remove these rather than comment them out if we're not going to be able to make them functional in this PR.

Reply:
Yep, I'm still messing around here -- I'm currently seeing whether it's feasible to update the older lists in CI, and I'll remove tests depending on how that pans out 🙂

Reply:
Done; I've rewritten the tests entirely (and enforced 100% coverage).